verification/VerifyTasks.wdl

```diff
@@ -133,4 +133,32 @@ task CompareCrais {
     memory: "2 GiB"
     preemptible: 3
   }
+}
+
+task CompareBams {
+
+  input {
+    File test_bam
+    File truth_bam
+  }
+
+  Float bam_size = size(test_bam, "GiB") + size(truth_bam, "GiB")
+  Int disk_size = ceil(bam_size * 4) + 20
+
+  command {
+    java -Xms3500m -jar /usr/picard/picard.jar \
+    CompareSAMs \
+          ~{test_bam} \
+          ~{truth_bam} \
+          O=comparison.tsv \
+          LENIENT_HEADER=true
```

Comment:
Does this fail if the bams are different? We may need to check the status of the call or check the contents of the output file

Reply:
Here is a link to a jenkins test that should have failed (was run without the LENIENT_HEADER=true arg to make sure the wdl fails and exits) : https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/4158/console


And here is a link to a test that passed (this test was run with the LENIENT_HEADER=true arg)  : https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/4174/console